### PR TITLE
chore(cli): run in zoom level asc order

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: MergerLogic/DataTypes/Data.cs [hunk 1]
```diff
@@ -34,6 +34,8 @@ public enum GridOrigin
 
     public abstract class Data<TUtilsType> : IData where TUtilsType : IDataUtils
     {
+        protected readonly int MaxZoomRead = 30;
```
Comment (asafMasa) [inline MergerLogic/DataTypes/Data.cs:37]: if it's used only in the S3 section - define it there

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

File: MergerLogic/DataTypes/Data.cs (hunk 1 above)
Comment (cl-shaharshavit-rnd) [reply MergerLogic/DataTypes/Data.cs:37]: why not const?
and why 30???
~72micron is insane resolution for map and un storable large
23 is already less then 1cm and more them we will EVER need for map

File: MergerLogic/DataTypes/S3.cs [hunk 2]
```diff
@@ -1,136 +1,173 @@
...
+            : base(container, DataType.S3, path, batchSize, grid, origin)
+        {
+            this._pathUtils = pathUtils;
+            this._bucket = bucket;
+            this._continuationToken = null;
+            this._endOfRead = false;
+            this._client = client;
+
+            // This should always happen after the definition of the client
+            this._zoomLevels = this.GetZoomLevels();
```
Comment (cl-shaharshavit-rnd) [inline MergerLogic/DataTypes/S3.cs:31]: why not converting it to array and then no reset is needed? its not enough data to be memory problem

**Review by cl-shaharshavit-rnd — CHANGES_REQUESTED** [review]

**Pull request merged**